client/src/main/webjar/toolbar/toolbar.scss

```diff
@@ -0,0 +1,3 @@
+.health-bam-toolbar-logo {
+    height: 100%;
```

Comment:
When in small-screen mode, the toolbar shrinks from 64px tall to 48px.  This keeps the logo image fitting in the toolbar.